securemeet.py

```
@@ -0,0 +1,907 @@
+                self.pipeline = Pipeline.from_pretrained(
+                    "pyannote/speaker-diarization-3.1", use_auth_token=None)
+            self.is_loaded = True
+            logger.info("Diarization pipeline ready")
+        except Exception as e:
+            logger.error(f"Diarization init failed: {e}")
+
+    def process_audio(self, audio_data: np.ndarray, sample_rate: int):
+        if not self.is_loaded:
+            return {"speakers": [], "segments": []}
+        try:
+            tmp = tempfile.NamedTemporaryFile(suffix=".wav", delete=False)
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)